__tests__/process-map.test.js

```diff
@@ -5,53 +5,54 @@ const tmpDir = `${__dirname}/tmp`;
 let { PptxUnitHelper } = require('../lib/helpers/unit-helper');
 
 describe('Presentation Module', () => {
-    test('should be able to load an existing pptx file', () => {
+    test('should be able to load an existing pptx file', async () => {
         try {
+            expect.assertions(3);
+
             let fulltemplateFilePath = `${__dirname}/fixtures/raci-matrix.pptx`;
             let presentation = new PPTX.Presentation({ templateFilePath: fulltemplateFilePath });
 
-            presentation.loadExistingPPTX(function(err) {
-                if (err) fail(err);
-
-                let slide1 = presentation.getSlide('slide1');
+            await presentation.loadExistingPPTX();
+            let slide1 = presentation.getSlide('slide1');
 
-                expect(slide1.content).toBeDefined();
-                expect(slide1.content).not.toBeNull();
+            expect(slide1.content).toBeDefined();
+            expect(slide1.content).not.toBeNull();
 
-                let shapeContent = slide1.getShapeRawContent(5);
+            let shapeContent = slide1.getShapeRawContent(5);
 
-                if (shapeContent) {
-                    if (shapeContent['p:txBody'] && shapeContent['p:txBody'][0]['a:p']) {
-                        if (shapeContent['p:txBody'][0]['a:p'][0]['a:r']) {
-                            shapeContent['p:txBody'][0]['a:p'][0]['a:r'][0]['a:t'] = "Shape's new text!";
-                            //shapeContent['p:txBody'][0]['a:p'][0]['a:endParaRPr'] = [{ $: { lang: 'en-US' } }]; // this MUST go last!
-                        } else {
-                            shapeContent['p:txBody'][0]['a:p'][0]['a:r'] = [
-                                { 'a:rPr': [{ $: { lang: 'en-US', smtClean: '0' } }], 'a:t': 'Mod our own shape' },
-                            ];
+            if (shapeContent) {
+                if (shapeContent['p:txBody'] && shapeContent['p:txBody'][0]['a:p']) {
+                    if (shapeContent['p:txBody'][0]['a:p'][0]['a:r']) {
+                        shapeContent['p:txBody'][0]['a:p'][0]['a:r'][0]['a:t'] = "Shape's new text!";
+                        //shapeContent['p:txBody'][0]['a:p'][0]['a:endParaRPr'] = [{ $: { lang: 'en-US' } }]; // this MUST go last!
```

Comment:
should we delete this comment if it's no longer needed?

Reply:
I'm keeping that there _just_ in case we run into that node ordering issue again. Idk why the issue disappeared for just that particular case. If doesn't pop up again in the next few days, I'll delete it.

Reply:
ok